temba/settings_common.py

```
@@ -985,6 +985,9 @@ def __getitem__(self, item):
     )
 }
 
+if not TESTING:
```

During testing we don't want this swallowing exceptions